Sources/SwiftInspectorVisitors/Tests/StructVisitorSpec.swift

```diff
@@ -49,10 +49,10 @@ final class StructVisitorSpec: QuickSpec {
               self.sut,
               overContent: content)
 
-            expect(self.sut.structs.first) == StructInfo(
-              name: "SomeStruct",
-              inheritsFromTypes: [],
-              parentTypeName: nil)
```

Comment:
Here and below I'm trying to follow the advice given in https://github.com/fdiaz/SwiftInspector/pull/56#discussion_r566425656